.circleci/bin/getversions

```diff
@@ -39,7 +39,7 @@ function get_latest_versions() {
       for (version in latest) {
           print latest[version]
       }
-  }' | sort
+  }' | sort -V
```

Comment:
TIL :heart:

Reply:
Yeah - I was busy coming up with convoluted, complex solutions and then Eralia found that!